### PR TITLE
feat: Release `Table` and `DataGrid` as stable

### PR DESCRIPTION
**Pull request by ling1726**
See title

### CODE REVIEW

File: packages/react-components/react-table/stories/DataGrid/Virtualization.stories.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
   VideoRegular,
 } from '@fluentui/react-icons';
 import { PresenceBadgeStatus, Avatar, useScrollbarWidth, useFluent } from '@fluentui/react-components';
-import { TableColumnDefinition, createTableColumn, TableCellLayout } from '@fluentui/react-components/unstable';
+import { TableColumnDefinition, createTableColumn, TableCellLayout } from '@fluentui/react-components';
```
Comment (miroslavstastny) [inline packages/react-components/react-table/stories/DataGrid/Virtualization.stories.tsx:12]: merge the imports to a single one

Comment (miroslavstastny) [reply packages/react-components/react-table/stories/DataGrid/Virtualization.stories.tsx:12]: there are multiple stories affected

**Review by miroslavstastny — APPROVED** [review]
Be more descriptive in the PR description, at least copy the title there :-)

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/Avatar-mount_base.html">1315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/Avatar-mount.html">1319</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/Button-mount_base.html">940</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/Button-mount.html">938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProvider-mount_base.html">1621</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProvider-mount.html">1640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProviderWithTheme-mount_base.html">637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProviderWithTheme-mount.html">634</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">581</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">597</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">633</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/InfoButton-mount_base.html">562</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/InfoButton-mount.html">564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/MakeStyles-mount_base.html">1898</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/MakeStyles-mount.html">1915</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/Persona-mount_base.html">3067</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/Persona-mount.html">3000</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/SpinButton-mount_base.html">2585</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26736/perf-test-react-components/SpinButton-mount.html">2552</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `64.042 kB`<br />`17.543 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `199.556 kB`<br />`55.412 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `35.006 kB`<br />`11.53 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.324 kB`<br />`2.129 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `135.751 kB`<br />`36.662 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `43.438 kB`<br />`11.933 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `133.853 kB`<br />`34.322 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `81.807 kB`<br />`20.399 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |   `81.236 kB`<br />`20.25 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/06b87ef7536c1bfe07c747a62a9168ea08ac6c5d'>06b87ef7536c1bfe07c747a62a9168ea08ac6c5d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Pull request merged**